net.veloren.airshipper.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: net.veloren.airshipper
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions : [
+  org.freedesktop.Sdk.Extension.rust-stable
+]
+command: airshipper
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --persist=.local/share/airshipper
+  - --share=ipc
+  - --device=dri
+build-options : {
+  append-path : "/usr/lib/sdk/rust-stable/bin",
+    build-args : [],
+      env : {
+        "CARGO_HOME" : "/run/build/airshipper/cargo",
+        "RUSTFLAGS" : "--remap-path-prefix =../",
+        "RUST_BACKTRACE" : "1"
+        }
+}
+
+modules:
+  - name : airshipper
+    buildsystem : simple
+    build-commands : 
+      - cargo --offline fetch --manifest-path Cargo.toml 
+      - cargo --offline build --bin airshipper --release 
+      - install -D -m 755 ./target/release/airshipper -t /app/bin/
+    post-install:
+      - install -D net.veloren.airshipper.desktop -t /app/share/applications/
+      - install -D net.veloren.airshipper.png -t /app/share/icons/hicolor/256x256/apps/
+      - install -D net.veloren.airshipper.metainfo.xml -t /app/share/metainfo/
+    sources:
+      - type: archive
+        url: https://github.com/Songtronix/Airshipper/archive/refs/tags/v0.4.2.tar.gz
+        sha256: a2a643f3a544e4689988233acf3152f22f09b982790e0a6d90f049589eaed87f
+      - type: patch
+        path: airshipper-client-add-desktop.patch
```

Comment:
Any reason for adding the desktop file as a patch instead of a proper file like the metainfo?

Reply:
The patch file was a remnant of the adaptation of the patches used to package Airshipper for other distributions.
Since I don't actually see a reason against having a proper file here, I added one. There is still a patch to allow building with stable Rust.